strongbox-cron/strongbox-cron-tasks/pom.xml

```diff
@@ -163,12 +163,10 @@
         </dependency>
 
         <dependency>
-            <groupId>org.apache.directory.server</groupId>
-            <artifactId>apacheds-core</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.directory.server</groupId>
-            <artifactId>apacheds-protocol-ldap</artifactId>
+            <groupId>com.unboundid</groupId>
+			<artifactId>unboundid-ldapsdk</artifactId>
+			<version>5.1.1</version>
+			<scope>test</scope>
```

Comment:
1. Cosmetic: This is not properly indented.
2. The version for this should only be defined in the `strongbox-parent`.

Reply:
Okay, fixed that, I thought I had removed it but apparently I hadn't.